pkg/scorecard.go

```diff
@@ -105,9 +105,25 @@ func RunScorecard(ctx context.Context,
 	defer repoClient.Close()
 
 	commitSHA, err := getRepoCommitHash(repoClient)
-	if err != nil || commitSHA == "" {
+
+	if errors.Is(err, sce.ErrorEmptyRepository) {
+		versionInfo := version.GetVersionInfo()
+		ret := ScorecardResult{
+			Repo: RepoInfo{
+				Name:      repo.URI(),
+				CommitSHA: commitSHA,
+			},
+			Scorecard: ScorecardInfo{
+				Version:   versionInfo.GitVersion,
+				CommitSHA: versionInfo.GitCommit,
+			},
+			Date: time.Now(),
+		}
+		return ret, nil
+	} else if err != nil || commitSHA == "" {
```

Comment:
do we still need the commitSHA comparison?

Reply:
i removed this. hopefully its not needed in some random other edge case